site-scrapers/HeywoodHealthcare.js

```diff
@@ -0,0 +1,28 @@
+const sites = require("../data/sites.json");
+
+const noAppointmentMatchString =
+    "All appointment types are private, none are available for scheduling.";
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(sites.HeywoodHealthcare.name + " starting.");
+    const webData = await ScrapeWebsiteData(browser);
+    console.log(sites.HeywoodHealthcare.name + " finished.");
+    return {
+        ...sites.HeywoodHealthcare,
+        ...webData,
+        timestamp: new Date(),
+    };
+};
+
+async function ScrapeWebsiteData(browser) {
+    const page = await browser.newPage();
+    await page.goto(sites.HeywoodHealthcare.signUpLink);
+    // Is the clinic open?
+    const alert = await (
```

Comment:
I just had a thought: this will cause an unhandled promise rejection if it's not there. We need to catch that rejection and return an empty string so that you rightly return that there is availability.

Reply:
@livgust Great catch! I'll make the change.